src/Navigation/Navigation.tsx

```diff
@@ -88,15 +89,24 @@ interface NavigationProps {
   id?: string;
   currentValue: Context;
   hostMap: HostMap;
+  buttonColor?: PropTypes.Color;
+  buttonStyle?: string;
+  triangleStyle?: string;
 }
 
 const Navigation: React.FC<NavigationProps> = ({
   currentValue,
   hostMap = {},
   id,
+  buttonColor = 'secondary',
+  buttonStyle,
+  triangleStyle,
 }) => {
   const classes = useStyles();
 
+  const button = buttonStyle || classes.button;
+  const triangle = triangleStyle || classes.triangle;
```

Comment:
I don't think `xxxStyle` is a good name. Classname would be better.
Also, you can use `clsx` to merge classes. So you can still use the default class and apply yours on top of it.

```
clsx( classes.button, buttonClassName )
```